### DISCUSSION TITLE
About 1.20.2

### DISCUSSION THREAD

**Discussion by vico93** [cc-tweaked/CC-Tweaked#1593] [General]
Ok, i will be *that* guy, but how much will need to be changed in CC:T code for a 1.20.2 release? The UI internal changes makes me suspicious that it will be a lot of stuff

**Comment by SquidDev** [comment]
> Ok, i will be that guy,

Please don't.

**Reply by vico93** [reply]
Sorry. Now i'm sad and embarassed 😳

**Comment by xAnavrins** [comment]
>Ok, i will be that guy

You have, in fact, been that guy, for every versions of the game mere days after their release.
